yarn-project/stdlib/src/keys/derivation.ts

```diff
@@ -121,27 +120,3 @@ export async function deriveKeys(secretKey: Fr) {
     publicKeys,
   };
 }
-
-// Returns shared tagging secret computed with Diffie-Hellman key exchange.
-async function computeTaggingSecretPoint(knownAddress: CompleteAddress, ivsk: Fq, externalAddress: AztecAddress) {
-  const knownPreaddress = await computePreaddress(await knownAddress.publicKeys.hash(), knownAddress.partialAddress);
-  // TODO: #8970 - Computation of address point from x coordinate might fail
-  const externalAddressPoint = await externalAddress.toAddressPoint();
-  const curve = new Grumpkin();
-  // Given A (known complete address) -> B (external address) and h == preaddress
-  // Compute shared secret as S = (h_A + ivsk_A) * Addr_Point_B
-
-  // Beware! h_a + ivsk_a (also known as the address secret) can lead to an address point with a negative y-coordinate, since there's two possible candidates
-  // computeAddressSecret takes care of selecting the one that leads to a positive y-coordinate, which is the only valid address point
-  return curve.mul(externalAddressPoint, await computeAddressSecret(knownPreaddress, ivsk));
-}
-
-export async function computeAppTaggingSecret(
-  knownAddress: CompleteAddress,
-  ivsk: Fq,
-  externalAddress: AztecAddress,
-  app: AztecAddress,
-) {
-  const taggingSecretPoint = await computeTaggingSecretPoint(knownAddress, ivsk, externalAddress);
-  return poseidon2Hash([taggingSecretPoint.x, taggingSecretPoint.y, app]);
-}
```

Comment:
This has been moved to `yarn-project/pxe/src/contract_function_simulator/tagging_utils.ts`